php-practice_1.php

```diff
@@ -1,41 +1,177 @@
 <?php
 // Q1 変数と文字列
+$name = '福倉';
+echo '私の名前は「' . $name . '」です。';
+// 出力結果：私の名前は「福倉」です。
 
 
 // Q2 四則演算
+$num = 5 * 4;
+echo $num;
+echo $num / 2;
+// 出力結果：20　10
 
 
 // Q3 日付操作
+date_default_timezone_set('Asia/Tokyo');
+echo "現在時刻は、" . date("Y年m月d日 H時i分s秒") . "です。";
+// 出力結果：現在時刻は、xxxx年xx月xx日 xx時xx分xx秒です。
 
 
 // Q4 条件分岐-1 if文
+$device = "mac"; 
+
+if ($device == "windows") {
+    echo "使用OSは、windowsです。";
+} else {
+    if ($device == "mac") {
+        echo "使用OSは、macです。";
+    } else {
+        echo "どちらでもありません。";
+    }
+}
+// 出力結果：使用OSは、macです。
 
 
 // Q5 条件分岐-2 三項演算子
+$age = 18;
+if ($age < 18) {
+    echo '未成年です。';
+} else {
+    echo '成人です。';
+}
+// 出力結果：未成年です。
 
 
 // Q6 配列
+$local = ['東京', '神奈川', '埼玉', '栃木', '千葉']; 
+echo $local[3] . 'と' . $local[4] . 'は関東地方の都道府県です。';
+// 出力結果：栃木県と千葉県は関東地方の都道府県です。
 
 
 // Q7 連想配列-1
+$kantou = [
+    "東京都"   => "新宿区",
+    "神奈川県" => "横浜市",
+    "千葉県"   => "千葉市",
+    "埼玉県"   => "さいたま市",
+    "栃木県"   => "宇都宮市",
+    "群馬県"   => "前橋市",
+    "茨城県"   => "水戸市"
+];
+
+foreach ($kantou as $city) {
+    echo $city . "<br>";
+}
```

Comment:
Q7
[must]
`<br>`という改行の仕方はPHPではなくHTMLの書き方なので、改行を表す特殊文字`\n`を使いましょう！
他に改行をしている箇所も同様に修正お願いします！

```suggestion
foreach ($kantou as $city) {
    echo $city . "\n";
}
```